modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -93,6 +115,274 @@ definition {
 		}
 	}
 
+	@priority = 4
+	test CanCreateEndpointWithSuccessAlert {
+		property portal.acceptance = "true";
+
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
+
+		task ("When I enter a path and Scope to create the API endpoint") {
+			APIBuilderUI.createAPIEndpoint(
+				description = "Description default",
+				path = "testendpoint2",
+				retrieveType = "Collection",
+				scope = "Company");
+		}
+
+		task ("Then I can see success alert") {
+			Alert.viewSuccessMessage();
+		}
+
+		task ("And Then I can see info of created endpoint with 'Get' method, scope, path") {
+			AssertElementPresent(
+				fieldLabel = "Method",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+
+			AssertElementPresent(
+				key_value = "GET",
+				locator1 = "APIBuilder#DROPDOWN_TOGGLE");
+
+			AssertElementPresent(
+				fieldLabel = "Scope",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+
+			AssertElementPresent(
+				fieldLabel = "Path",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+		}
+	}
+
+	@priority = 4
+	test CanSeeAllFieldsInCreationModal {
+		property portal.acceptance = "true";
+
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
+
+		task ("When I click 'Add API Endpoint' button") {
+			Click(locator1 = "APIBuilder#ADD_API_ENDPOINT");
+		}
+
+		task ("Then fields Scope *, Path * and Description are present") {
+			AssertElementPresent(
+				fieldLabel = "Scope",
+				locator1 = "FormViewBuilder#FIELD_FROM_FIELDSET");
```

Comment:
Could you try to assert 'Scope*' and 'Path*', because for the step we want to assert scope and path are required field